### PR TITLE
Fix integration tests

### LINKED ISSUE #1199: Fix integration tests
Account for warning from upstream packages:

```
partial match of 'just' to 'justification'
```

### PR DESCRIPTION
**Pull request by edelarua**
# Pull Request

Fixes #1199

### CODE REVIEW

File: tests/testthat/setup.R [hunk 1]
```diff
@@ -28,7 +28,10 @@ expect_snapshot_ggplot <- function(title, fig, width = NA, height = NA) {
 
   name <- paste0(title, ".svg")
   path <- tempdir()
-  suppressMessages(ggplot2::ggsave(name, fig, path = path, width = width, height = height))
+  withr::with_options(
+    opts_partial_match_old,
```
[the remark this answers is not in the document]
Comment (edelarua) [reply tests/testthat/setup.R:32]: Pawel added it in recently here:
https://github.com/insightsengineering/tern/blob/9a3ac37aa7a308d47cab411cdfa5dc697e82c90e/tests/testthat/setup-options.R#L4

**Review by Melkiades — APPROVED** [review]
Lgtm!! Thanks Emily ;)

**Comment by pawelru** [comment]
Sorry for jumping into the merged PR - would you be able to give some hints why we exclude? I tried to provide some additional explanation why. For instance, `just`-`justify` issue stems from this: https://github.com/r-lib/gtable/pull/94

This would help in the future when one will be trying to remove these excludes.

**Comment by edelarua** [comment]
> Sorry for jumping into the merged PR - would you be able to give some hints why we exclude? I tried to provide some additional explanation why. For instance, `just`-`justify` issue stems from this: [r-lib/gtable#94](https://github.com/r-lib/gtable/pull/94)
> 
> This would help in the future when one will be trying to remove these excludes.

Yes, of course - we are excluding due to the following message:
```
partial match of 'just' to 'justification'
```
This stems from the PR you've mentioned from `gtable` ([r-lib/gtable#94](https://github.com/r-lib/gtable/pull/94)), which is affecting `ggplot2` and all downstream packages, including `tern`. Once this fix is implemented, we should be able to remove all of the excludes added in this PR.

**Pull request merged**